xs2a-adapter-aspsp-registry/src/test/java/de/adorsys/xs2a/adapter/registry/LuceneAspspRepositoryTest.java

```diff
@@ -196,6 +197,70 @@ public void findLike() {
         assertThat(found).hasSize(2);
     }
 
+    @Test
+    public void findLikeShouldFollowTheStrictOrder() {
```

Comment:
It would be nice to state the actual order instead of the generic adjective.

Reply:
Renamed, thanks

Reply:
Perhaps I didn't make myself clear. I want the expected order to be captured in the test name. This way somebody who will need to fix a failure of this test in the future will know the intended order rather than trying to guess the order from the implementation.

Reply:
Fixed, thanks. Please, take a look at the new test name.